### PR TITLE
62 upgrade to go 119

### PR DESCRIPTION
**Pull request by amirhnajafiz**
@1995parham, @mehditeymorian jan could you please review this PR.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -6,8 +6,14 @@ linters:
   enable-all: true
   disable:
     # it should improve to support more known patterns
-    - varnamelen
+    - varnamelen
```
Comment (1995parham) [inline .golangci.yml:9]: I think there is a redundant whitespace here.

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
       - name: set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.18
+          go-version: 1.19
```
Comment (1995parham) [inline .github/workflows/go.yml:19]: Can we use 1.20?

File: server.go [hunk 3]
```diff
@@ -54,7 +54,7 @@ type Server interface {
 func NewServer(address string, topics []string, config *ServerConfig) (Server, error) {
 	config = processServerConfig(config)
 
-	listener, err := quic.ListenAddr(address, config.TLSConfig, nil)
+	listener, err := quic.ListenAddr(address, config.TLSConfig, nil) //nolint:typecheck
```
Comment (1995parham) [inline server.go:57]: Ignoring `typecheck`? Can you please share the error here?

**Review by 1995parham — CHANGES_REQUESTED** [review]

File: .github/workflows/go.yml (hunk 2 above)
Comment (amirhnajafiz) [reply .github/workflows/go.yml:19]: @1995parham jan, it's look like ```quic-go``` has some problems on ```v1.20```.

```shell
quic-go@v0.31.1/internal/qtls/go120.go:5:13: cannot use "The version of quic-go you're using can't be built on Go 1.20 yet.
```

File: server.go (hunk 3 above)
Comment (amirhnajafiz) [reply server.go:57]: It is a mistake. I will fix it.

**Review by 1995parham — APPROVED** [review]

**Pull request merged**